frontend/src/app/spot/styles/sass/components/list.sass

```diff
@@ -14,32 +14,68 @@
       @include spot-body-big
 
       display: flex
-      align-items: center
+      align-items: flex-start
       justify-content: flex-start
-      padding: $spot-spacing-0_5
+      padding: $spot-spacing-0_5 $spot-spacing-1
       background-color: transparent
       margin: 0
       border: 0
       cursor: pointer
 
       &:hover
-        background-color: $spot-color-main-light
+        background-color: $spot-color-basic-gray-6
 
       &_disabled
         color: $spot-color-basic-gray-3
         cursor: default
 
+    &-floating-wrapper
+      position: relative
+      display: flex
+
+      &:hover,
+      &:focus-within
+        background-color: $spot-color-basic-gray-6
+
+    &-floating-actions
+      opacity: 0
+      position: absolute
+      right: 0
+      height: 100%
+      padding: $spot-spacing-0_5 $spot-spacing-1
+      background: linear-gradient(90deg, rgba($spot-color-basic-gray-6, 0) 0%, $spot-color-basic-gray-6 11.98%)
```

Comment:
11.98% is a very random value.. can't it be 12%?

Reply:
it seems random and was directly taken from the figma sketches. Not sure, what's the background of this value, @psatyal ?

Reply:
Hi @HDinger and @Kharonus.

Could you show me where the 11,98% is in Figma? I can't find it when I look (it's 100% to 0%, but since @marcalcobe worked on this, perhaps it's elsewhere and I don't know).

Reply:
As for mobile, yes, we'll have to work on describing mobile behaviour if that's not done yet. It would surely involve something a dots menu, as @Kharonus suggested in [another comment ](https://github.com/opf/openproject/pull/10762#issuecomment-1149870629).